builds/serverless-config-pack/serverless-java.yml

```diff
@@ -87,28 +87,30 @@ functions:
           - ","
           - ${file(deployment-env.yml):subnetIds}
 
+#Start:resources
 resourcesDisabled:
   Resources:
     #Start:isDynamoDbEnabled
     customEventRole: ${file(policyFile.yml):eventSubscribed}
     #End:isDynamoDbEnabled
-    #Start:isSqsEnabled
+    #Start:isSQSEventEnabled
     customEventRole: ${file(policyFile.yml):sqsRole}
-    #End:isSqsEnabled
-    #Start:isSqsEnabled
+    #End:isSQSEventEnabled
+    #Start:isSQSEventEnabled
```

Comment:
Why is #Start:isSQSEventEnabled added again?

Reply:
The comment section added here is to remove the section when it is not using. The comment previously added was   #Start:isSqsEnabled   #End:isSqsEnabled But in lambda , used #Start:isSQSEventEnabled   #End:isSQSEventEnabled  to remove.  In order to fix that, I had renamed the comment.

Reply:
I'm looking at code fragment below - 
`#End:isSQSEventEnabled`   
`#Start:isSQSEventEnabled`

Will the below work - ?

    `#Start:isSQSEventEnabled`
    `customEventRole: ${file(policyFile.yml):sqsRole}`
    `#Start:isSqsResourceNotExist`
    `{event_source_sqs}:`
      `Type: AWS::SQS::Queue`
      `Properties:`
        `QueueName: {event_source_sqs}`
        `VisibilityTimeout: 165`
    `#End:isSqsResourceNotExist`
    `#End:isSQSEventEnabled`

Reply:
Oh ok understood your point. 
The SQS flow is , first  will add the sqs queue name in the resource part and will deploy, if the queue not exists, serverless will create the resource - by using the serverless yml file and will attach the lambda arn to the queue. Deployment will be success in this case.
2. If the queue already exists - Serverless will add the queue name in the resource part and will deploy, this time we will get an error while deploying saying resource already exists. So I will capture that error and remove the resource part from the serverless yml file and will redploy. So for removing the resource part only, I had added that comment section, since we need the custom role for both the case